app/schema.py

```diff
@@ -3,6 +3,8 @@
 from pydantic import BaseModel, constr, Extra
 
 
+# TODO add pydantic to requirements.txt
```

Comment:
Why not add Pydantic to the requirements.txt instead?

- go to the terminal in the IDE, make sure your virtual environment for the project is active
- enter the command `pip list`
- find pydantic
- I would enter it in the format of `pydantic~=<version number>`

Reply:
I can add it if needed. I wasn't sure if it was out of the scope of this branch to add those changes.

Reply:
I went a head an added it. It's on the most recent change.